### PR TITLE
chore: add python repo templates

### PR DESCRIPTION
**Pull request by busunkim96**
Add some python repo templates via synthtool to enable the usual release flow (release-please and Kokoro release builds). 

Needs corresponding kokoro config change internally.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,12 +1,7 @@
-Before you open a pull request, note that this repository is forked from [here](https://github.com/docascode/sphinx-docfx-yaml/).
-Unless the issue you're trying to solve is unique to this specific repository, 
-please file an issue and/or send changes upstream to the original as well.
```
Comment (dandhlee) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: Could you preserve these 3 lines?

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# How to Contribute
```
Comment (dandhlee) [inline .github/CONTRIBUTING.md:1]: we already have CONTRIBUTING.md in the top directory, should we move that under `.github` directory?

File: setup.py [hunk 3]
```diff
@@ -30,7 +30,7 @@
     version='1.2.76',
     author='Eric Holscher',
     author_email='eric@ericholscher.com',
-    url='https://github.com/ericholscher/sphinx-docfx-yaml',
+    url='https://github.com/googleapis/sphinx-docfx-yaml',
```
Comment (dandhlee) [inline setup.py:33]: Could you update the file with the following?
```
import codecs
import setuptools

name = 'gcp-sphinx-docfx-yaml'
description = 'Sphinx Python Domain to DocFX YAML Generator'
version = '0.1.0'
dependencies = [
    'PyYAML',
    'wheel>=0.24.0',
    'sphinx',
    'unidecode'
]

packages = setuptools.find_packages('.', exclude=['tests'])

extra_setup = dict(
setup_requires=['pytest-runner'],
tests_require=['pytest', 'mock'],
)

setuptools.setup(
    name=name,
    version=version,
    description=description,
    author='Google LLC',
    author_email='dandhlee@google.com',
    license='Apache 2.0',
    url='https://github.com/googleapis/sphinx-docfx-yaml',
    package_dir={'': '.'},
    packages=packages,
    install_requires=dependencies,
    include_package_data=True,
    zip_safe=False,
    **extra_setup
)
```

**Review by dandhlee — CHANGES_REQUESTED** [review]
Thank you so much for doing this! Just few minor changes would be great!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (busunkim96) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: Yep I'll add it to the `excludes` so it doesn't get overwritten

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (busunkim96) [reply .github/CONTRIBUTING.md:1]: Same comment as above, I'll just exclude this from the copy

**Review by dandhlee — APPROVED** [review]
Thank you!

**Pull request merged**